examples/src/main/java/org/apache/spark/examples/ml/JavaNGramExample.java

```diff
@@ -55,16 +55,12 @@ public static void main(String[] args) {
 
     Dataset<Row> wordDataFrame = spark.createDataFrame(data, schema);
 
-    NGram ngramTransformer = new NGram().setInputCol("words").setOutputCol("ngrams");
+    NGram ngramTransformer = new NGram().setN(2).setInputCol("words").setOutputCol("bigrams");
```

Comment:
I really only think that the param `N` should be set explicitly.  Looking back, changing the column name was not necessary, let me change that back.